Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -24,14 +24,17 @@
  *  Rev B  2 JUN 2017
  *
  *  Converted to Arduino pin numbering
+ *  at90usb1286 to arduino pin mapping is here:
+ *     https://labitat.dk/wiki/Panelolu_and_Printrboard_the_easy_way
```

Comment:
Note that Marlin has switched to a unified Arduino-style pin mapping under `fastio.h`, including for AT90USB processors. All code uses FastIO macros —such as `READ`, `WRITE`, etc.— allowing us to follow this mapping consistently. We do not use the so-called "Marlin mapping" or "fastio mapping" which numbers A0-A7 as 0-7, B0-B7 as 8-15, etc. We have killed that mapping.